src/components/cart/cart-actions/CartActions.tsx

```diff
@@ -0,0 +1,19 @@
+import './CartActions.scss';
+
+import React from 'react';
+
+type Props = {
```

Comment:
**suggestion:** i suggest naming this props to be more significant to the component you are using it. In case you need them exported to be used in some other place in your code you don't want a hundred `Props` types